test/unit/apps/gro/controllers/address.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+const BaseController = sinon.stub();
+const ErrorController = require('hof').controllers.error;
+
+const AddressController = proxyquire('../../../../../apps/gro/controllers/address', {
+  'hof': {
+    controllers: {
+      base: BaseController,
+      error: ErrorController
+    }
+  }
+});
+
+describe('apps/gro/controllers/address', () => {
+
+  let controller;
```

Comment:
The new instance of a controller needs to be made before each test (line 53). I suppose you could do `const controller = new AddressController(options);`. However, I think that it is better practise to declare the variable here and then initialise it further down. This is also how it's done in the other tests too.